LBH.AdultSocialCare.Api/V1/Boundary/DayCarePackageBoundary/Request/DayCarePackageForUpdateRequest.cs

```diff
@@ -24,6 +24,6 @@ public class DayCarePackageForUpdateRequest
         [Required] public bool? EscortNeeded { get; set; }
         [Required] public int? TermTimeConsiderationOptionId { get; set; }
         [Required, GuidNotEmpty] public Guid UpdaterId { get; set; }
-        [Required, GuidNotEmpty] public Guid StatusId { get; set; }
+        [Required, GuidNotEmpty] public int StatusId { get; set; }
```

Comment:
The GuidNotEmpty attribute should be removed here

Reply:
My Bad!